main.py

```diff
@@ -2,6 +2,58 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # create
+    genres_names_to_create = ["Western", "Action", "Dramma"]
+    for genre_name in genres_names_to_create:
+        Genre.objects.create(
+            name=genre_name
+        )
+
+    actors_names_to_create = [
+        "George Klooney", "Kianu Reaves", "Scarlett Keegan",
+        "Will Smith", "Jaden Smith", "Scarlett Johansson"
+    ]
+    for actor_name in actors_names_to_create:
+        actor_name_dict = {"first_name": actor_name.split()[0],
+                           "last_name": actor_name.split()[1]}
```

Comment:
why not create a list of tuples? you will not need to use split later

Reply:
i thought about the idea that we input the full name and next create an actor. So, the creation of a list of tuples just adds one step more

Reply:
Why it will add one more step ? Your data will be already separated be first and last name in each tuple . For example --- >
```
actors_data = [
    ( first_name, last_name),
    (... , ... )
]
```
Then --- >

`for first_name , last_name in actors_data:`